tools/install_module.sh

```diff
@@ -0,0 +1,18 @@
+
+echo Install $1 Module
+
+# Build Module Dependencies
+cd $MODULE_DIR
+composer update --prefer-dist --no-interaction --no-progress
+cd $TRAVIS_BUILD_DIR
+
+# Move Module Contents to Install Folder
+echo Move Module Contents to Prestashop Modules Directory
+mkdir     $TRAVIS_BUILD_DIR/modules/$1
+cp -Rf    $MODULE_DIR/*              $TRAVIS_BUILD_DIR/modules/$1/      
+
+# Change Default Language Code to ISO format
+mysql -D prestashop -e "UPDATE ps_lang SET language_code = 'fr-fr' WHERE ps_lang.language_code = 'fr';"
+
+# Enable the Module
+mysql -D prestashop -e "INSERT INTO ps_module ( name ,  active ,  version ) VALUES ('coinbase', 1 , 'test');"
```

Comment:
What about using the console?

```
php bin/console prestashop:module install coinbase
```

Reply:
Good idea, but that command was introduced in which version now?

Reply:
PrestaShop 1.7.2.0: https://devdocs.prestashop.com/1.7/modules/testing/